src/RoleObjectFactory.js

```diff
@@ -364,6 +364,7 @@ function createAccessibilityObjectForRole(config) {
       parent.accessible.addChildAt(displayObject, containerIndex);
     } else {
       parent.accessible.addChild(displayObject);
+      containerIndex = _.findIndex(parent.children, child => child === displayObject);
```

Comment:
This should be earlier in the function so that the index can be applied on line 364. I would recommend moving this to just after line 90 and before any changes for addressing my comment on that line.  Though since it's possible to have an AccessibilityObject instance that isn't presently in the accessibility tree, this should also probably be wrapped in an if-statement.  And since it's the same condition as my previous comment, these comments can be combined so that the following should cover them:
```
if (parent) {
  containerIndex = _.findIndex(parent.children, child => child === displayObject);
  parent.removedChild(displayObject);
}
```